nix/release.nix

```diff
@@ -0,0 +1,56 @@
+{ buildEnv
+, ghr
+, writeShellScriptBin
+, postgrest
+, postgrestDocker
+, runCommand
+}:
+let
+  version = postgrest.version;
```

Comment:
Trying to understand a bit. How `postgrest` gets the `version` attribute?

Reply:
Yes, that's not so obvious - it comes from `postgrest.cabal` via `callCabal2nix`. As I understand, most derivations in nix have that attribute, next to `pname` for the package name, for example.

Reply:
Oh, I see.

I was trying to do a `builtins.trace` to understand that a bit more. But somehow doing:

```
nix-env -iA release -f default.nix
```
Makes my laptop freeze :fearful:(twice now). Never had this before, is there some evaluation in the nix file that could be expensive?

Reply:
(After adding a `-v` to the command, I'm seeing it's evaluating a lot of files, maybe it's rebuilding the static bin somehow. I'll see if cachix is set).

Reply:
Hm, that's weird - everything froze? Evaluating the Nix expression without any cache is probably not cheap, but it should not break a whole machine...

Reply:
Yeah, the xserver froze and I couldn't access the other ttys(f1, f2, etc). But it's no biggie since I can use my desktop machine for evaluating the expression if no cache is available.